data/en/4.0.dict

```diff
@@ -11375,7 +11375,7 @@ $ USD.c US$.c C$.c AUD.c AUD$.c HK.c HK$.c
 "&": G- & {Xd- & G-} & G+;
 
 % The double-single-quote is converted to a single single quote by m4.
-"’" "''": YP- & (({AL-} & {@L+} & (D+ or DD+)) or [[<noun-main-x>]] or DP+);
+"’" "'": YP- & (({AL-} & {@L+} & (D+ or DD+)) or [[<noun-main-x>]] or DP+);
 
```

Comment:
the change above is immediately contradicted by the text in the line above it: viz: "% The double-single-quote is converted to a single single quote by m4."